helpers/handlebars/helpers/index.js

```diff
@@ -133,6 +133,9 @@ module.exports = {
 		}
 		return opts.inverse(this);
 	},
+	userHasRole: (role, opts) => {
+		return !!opts.data.local.currentUser.roles.find((r) => r.name === role);
```

Comment:
check that logic is nowhere else used in the client and replace it using that helper ... is that function relly nowhere else already defined?

Reply:
@tofixx updated it

Reply:
great work @adrianjost 

love that all this **** is gone now :)